azdo/azdo.go

```diff
@@ -79,6 +79,13 @@ func SetPipelineVariable(name, value string) {
 	fmt.Printf("##vso[task.setvariable variable=%v]%v\n", name, value)
 }
 
+// UploadBuildSummary uses an AzDO logging command to upload a build summary. When used on a
+// markdown file, the rendered document shows up on the build page in an "Extensions" tab. The path
+// must be a full path.
+func UploadBuildSummary(path string) {
```

Comment:
This is more generic than build, can be used to upload anything. Consider naming `UploadSummary`

Reply:
Hmm, what I originally had in mind for the name was that it's just specifying "AzDO build" for a little clarify at the point where the func is used. Similar to `GetBuildWebURL`. (But `upload a build summary` in the comment ruins that.)

For `task.setvariable` I had the same thing in mind, but I called it `SetPipelineVariable(name, value string)`. Not consistent vs. `Build`.

Maybe it'd get the point across better to make it explicit and stick more closely to the `task.setvariable`/`task.uploadsummary` names: `LogCmdSetVariable`, `LogCmdUploadSummary`. It would also make it easier to recognize that this isn't a network call. (The lack of `ctx context.Context` and return value seems odd if you think it's a network call.) I'm doing this rename.